### PR TITLE
Bindings for Terminal endpoints

### PR DESCRIPTION
**Pull request by daz-stripe**
Bindings for Terminal endpoints
- Connection Tokens
- Locations
- Readers

### CODE REVIEW

File: stripe/api_resources/terminal/reader.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from __future__ import absolute_import, division, print_function
+
+from stripe.api_resources.abstract import CreateableAPIResource
+from stripe.api_resources.abstract import UpdateableAPIResource
+from stripe.api_resources.abstract import ListableAPIResource
+
+
+class Reader(CreateableAPIResource, ListableAPIResource,
+           UpdateableAPIResource):
```
Comment (ob-stripe) [inline stripe/api_resources/terminal/reader.py]: Same here.

File: stripe/api_resources/terminal/location.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from __future__ import absolute_import, division, print_function
+
+from stripe.api_resources.abstract import CreateableAPIResource
+from stripe.api_resources.abstract import UpdateableAPIResource
+from stripe.api_resources.abstract import ListableAPIResource
+
+
+class Location(CreateableAPIResource, ListableAPIResource,
+           UpdateableAPIResource):
```
Comment (ob-stripe) [inline stripe/api_resources/terminal/location.py]: Need to align this line with `CreatableAPIResource` to satisfy the linter.

**Comment by brandur-stripe** [comment]
LGTM!

**Comment by brandur-stripe** [comment]
Released as 2.10.0.

**Pull request merged**